CHANGES_NEXT_RELEASE

```diff
@@ -1,4 +1,5 @@
 - Add: -mqttTimeout (env var ORION_MQTT_TIMEOUT) for MQTT broker connection timeout
 - Fix: queue workers deadlock when MQTT broker was not responding to connection attemp
 - Fix: Ngsiv2-AttrsFormat header set to "normalized" in NGSI patching custom notification
-- Fix: Orion should return error and refuse to start when port is negative (3875)
+- Fix: Orion should return error and refuse to start when port is negative (#3875)
+- Fix: logLineMaxSize and logLineMaxSize in the log admin REST API (#3707)
```

Comment:
Little typo:

```suggestion
- Fix: logLineMaxSize and logInfoPayloadMaxSize in the log admin REST API (#3707)
```

Reply:
Fixed in [24de8d9](https://github.com/telefonicaid/fiware-orion/pull/4289/commits/24de8d9e544ab1ddba1af46cdb70ded49bb85e2e)